data/incarnations/tolIncarnationLineages.ttl

```diff
@@ -2576,7 +2552,7 @@ told:_IncarnationLineage_62
 		"ཚེ་སྨོན་གླིང་།"@bo
 		;
 	gist:isConnectedTo
-		<https://w3id.org/treasuryoflives/ns/data/tol/_Place_724,TOL36> ,
+		<https://w3id.org/treasuryoflives/ns/data/tol/_Place_724;TOL36> ,
```

Comment:
Here is a case where the CSV has semi-colon without a following space. The TARQL looks for a semi-colon with a following space. This will need to be standardized throughout a file. I've fixed this as well.